src/apps/ipv6/nd_light.lua

```diff
@@ -77,8 +77,12 @@ local function check_ip_address(ip, desc)
 end
 
 function nd_light:new (arg)
-   local conf = arg and config.parse_app_arg(arg) or {}
-   local o = nd_light:superClass().new(self)
+   local arg = arg and config.parse_app_arg(arg) or {}
+   local conf = {}
+	for k,v in pairs(arg) do
+		conf[k] = v
+	end
+	local o = nd_light:superClass().new(self)
```

Comment:
@capr Not exactly a fan of this code, can you change this to simply copy stuff into `o._config` (preferably without the loop)?